vllm/model_executor/layers/activation.py

```diff
@@ -159,6 +159,21 @@ def forward_cuda(self, x: torch.Tensor) -> torch.Tensor:
     # def forward_xpu(self, x: torch.Tensor) -> torch.Tensor:
 
 
+class ReLUSquaredActivation(CustomOp):
+    """
+    Applies the relu^2 activation introduced in https://arxiv.org/abs/2109.08668v2
+    """
```

Comment:
I think you might also need to add this op to CPU similar to what I did [here](https://github.com/vllm-project/vllm/pull/5717) previously with `QuickGELU`.